frontend/src/modules/fluentoriginal/components/RichString.test.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import sinon from 'sinon';
+
+import RichString from './RichString';
+
+
+const ORIGINAL = 'song-title = Hello\n    .genre = Pop\n    .album = Hello and Good Bye';
+
+const ENTITY = {
+    original: ORIGINAL,
+};
+
+describe('<RichString>', () => {
+    it('renders correct value and each attribute of original input', () => {
+        const wrapper = shallow(<RichString
+            entity = { ENTITY }
+        />);
+
+        expect(wrapper.find('RichString')).toBeDefined();
```

Comment:
I believe that's redundant (also in tests below), because it's pretty obvious it will be true given the initialization of the `wrapper` above.